salt/common/files/log-rotate.conf

```
@@ -11,13 +11,13 @@
 /opt/so/log/soctopus/*.log
 /opt/so/log/curator/*.log
 /opt/so/log/fleet/*.log
-/opt/so/log/suricata/*.log
+/opt/so/log/suricata/suricata.log
```

This was a good idea, but it makes more sense to instead remove the additional `logrotate` config for stats.log and keep the wildcard for this directory. `surirotate` was a legacy cron job that we missed when adding additional log rotation to Security Onion.
```suggestion
/opt/so/log/suricata/*.log
```

In addition to the above:
- Remove `salt/suricata/cron/surirotate`
- Remove `salt/suricata/files/suri-rotate.conf`
- Remove lines 84-88 and 177-190 from `salt/suricata/init.sls`